### PR TITLE
added in some line in Hello.java

### PR DESCRIPTION
**Pull request by alikny**

### CODE REVIEW

**Comment by alikny** [comment]
Mates, I added some code into the |Hello.java.. please comment on that code ..:))

**Comment by hasantarikkarakus** [comment]
Hi Ali, I liked your commit :)

**Comment by alikny** [comment]
> Hi Ali, I liked your commit :)

but i do not like your awesome comment  :))  you can make it better :)

File: src/test/java/Hello.java [hunk 1]
```diff
@@ -1,5 +1,10 @@
 public class Hello {
     public static void main(String[] args) {
         System.out.println("hello");
+
+        //this line added by alisbranch in local and commit  pushed
+
+
+
```
Comment (alikny) [inline src/test/java/Hello.java:8]: i added that part

**Comment by hasantarikkarakus** [comment]
ok, so please do the better one, under my pull request :)

**Comment by muhsintamturk** [comment]
You are very good in joint project. welldone

**Comment by alikny** [comment]
ok, lets merge it

**Comment by alikny** [comment]
wqwefasdfasfdaf

**Pull request closed without merging**